doc/schemas/node_openapi.json

```diff
@@ -907,7 +908,7 @@
   "info": {
     "description": "This API provides public, uncredentialed access to service and node state.",
     "title": "CCF Public Node API",
-    "version": "4.1.0"
+    "version": "4.2.0"
```

Comment:
Does this belong here?

Reply:
Weirdly yes, because of the change of enum above. We expose the consensus state through a REST API for debugging/ops purposes, and that's specified in this file that although checked in is in fact generated automatically from our types, and checked by a test. Making changes to the file requires bumping up the version, to communicate the change to a potential user. This is also enforced by the test.